README.md

```diff
@@ -109,8 +136,21 @@ jq.run(filter, jsonPath, options)
 
 ### path to jq binary
 
-By default, the `jq` binary installed with the package is used. If you have special needs or want to use another binary in a different
-path you can set the environment variable `JQ_PATH` to override the binary path.
+By default, the `jq` binary installed with the package is used. You can override this in several ways:
+
+1. **`.npmrc` configuration** (recommended for project-level settings):
+   ```bash
+   jq-path=jq
+   # or
+   jq-path=/usr/local/bin/jq
+   ```
+
+2. **Environment variable** (for runtime overrides):
+   ```bash
+   export JQ_PATH=/path/to/jq
+   ```
+
+The priority order is: `JQ_PATH` environment variable → `.npmrc` configuration → downloaded binary.
```

Comment:
Keep the documentation as before, and mention the possibility to add "jq_path" in npmrc or package.json.